### ISSUE TITLE
Delayed request body

### ISSUE THREAD

**Issue by ravi160822** [swagger-api/swagger-ui#8246]
[Screen Recording 2022-10-21 at 4.49.03 AM.zip](https://github.com/swagger-api/swagger-ui/files/9844622/Screen.Recording.2022-10-21.at.4.49.03.AM.zip)

I update the rank, but it gets displayed late, it gets displayed only after I click in the swagger ui.
The rank should change as soon as I hit execute.

python 3.10.7, FastAPI 0.85, macOS, Safari

**Comment by robert-hebel-sb** [comment]
We are doing some housekeeping on our issue tracker. This issue has been inactive for over a year and we have not been able to tackle it yet.

Is this still important to you? If so, please leave a comment – we'd love to know if you're still affected. If we don't hear back in 7 days, we'll close this issue to tidy up the tracker.

Thank you for your understanding!

**Comment by robert-hebel-sb** [comment]
No response received after 7 days. Closing this issue to keep the tracker tidy. If you're still affected, please reopen or create a new issue with updated details.
